CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # Changelog for Thoth's micropipenv
 
+## [1.3.0] - 2022-April-08 - Fridolin Pokorny <fridolin@redhat.com>
+
+### Added
+
+* Improve poetry → requirements and handling of transitive deps
+  Contribution thanks to @frenzymadness
+
+* Improve handling of poetry → requirements
```

Comment:
Isn't this point already included in the previous one?

Reply:
Thanks for the review, adjusted 👍🏻